test/test_nn.py

```diff
@@ -19910,7 +19920,8 @@ def _nll_loss_helper(self, input_size, reduction, expected, device):
 
         output = F.nll_loss(input, target, reduction=reduction)
         # TODO(#38095): Replace assertEqualIgnoreType. See issue #38095
-        self.assertEqualIgnoreType(output, expected)
+        # self.assertEqualIgnoreType(output, expected)
+        self.assertEqual(output, expected.to(dtype=output.dtype))
```

Comment:
If `output.dtype` always the same, put the value here. Otherwise it's better to just use `exact_dtype=False` parameter for `assertEqual` instead of `to`.